docs/v2-release-notes.md

```diff
@@ -5,8 +5,13 @@ This page summarizes changes for each major update in Timeplus Enterprise v2 and
 ## Jul 22, 2024
 
 ### Timeplus Proton v1.5.14
-  * New feature: Remote user-defined functions (UDFs) are now supported.
+  * Remote user-defined functions (UDFs) can now be created via SQL.
     * Example: `CREATE REMOTE FUNCTION ip_lookup(ip string) RETURNS string URL 'https://abc.lambda-url.us-west-2.on.aws/'`
+  * Timeplus-native-jdbc is updated (v2.0.2):
```

Comment:
Timeplus-native-jdbc is a different repo, not part of Timeplus Proton or Timeplus Enterprise. If we have to chose one, it belongs to Timeplus Enterprise.. Should not be put between Proton 1.5.13 and 1.5.14